stubs/unidiff/__init__.pyi

```diff
@@ -0,0 +1,14 @@
+from typing import List
```

Comment:
Is this here because unidiff doesn't provide its own types? Is there a reason this is in a folder called "stubs"?

Reply:
Never mind, found some documentation discussing this.